src/wobbly-effect.cpp

```diff
@@ -77,6 +77,8 @@ endless_shell_fx_wobbly_lie_about_paint_volume (ClutterEffect    *effect,
                                                                                         volume);
 }
 
+#define DAMAGE_RECTANGLE_EXPAND_FACTOR 8
```

Comment:
Ouch, this feels a bit like stabbing in the dark. Are you sure that we're not papering over some other issue with this, such as using the wrong actor to calculate the paint volume, or a bug in the wobbly model calculations?
I would at least like to know where those 8px come from, and have a clear understanding, encoded in a comment, of why we can't trust the numbers we get from Clutter/libwobbly.